enterprise/payara.eecommon/nbproject/org-netbeans-modules-payara-eecommon.sig

```diff
@@ -1231,16 +1231,16 @@ supr java.lang.Enum<org.netbeans.modules.payara.eecommon.api.config.JavaEEModule
 hfds LOGGER,j2eeModuleTypeToValue,moduleType
 
 CLSS public abstract org.netbeans.modules.payara.eecommon.api.config.PayaraConfiguration
-cons protected init(org.netbeans.modules.j2ee.deployment.devmodules.api.J2eeModule,org.netbeans.modules.payara.eecommon.api.config.J2eeModuleHelper,org.netbeans.modules.payara.tooling.data.PayaraVersion) throws org.netbeans.modules.j2ee.deployment.common.api.ConfigurationException
-cons protected init(org.netbeans.modules.j2ee.deployment.devmodules.api.J2eeModule,org.netbeans.modules.payara.tooling.data.PayaraVersion) throws org.netbeans.modules.j2ee.deployment.common.api.ConfigurationException
+cons protected init(org.netbeans.modules.j2ee.deployment.devmodules.api.J2eeModule,org.netbeans.modules.payara.eecommon.api.config.J2eeModuleHelper,org.netbeans.modules.payara.tooling.data.PayaraPlatformVersionAPI) throws org.netbeans.modules.j2ee.deployment.common.api.ConfigurationException
```

Comment:
Keep the original constructor and just introduce another one. The old constructor shall delegate to the new one, I guess.